Devscord.DiscordFramework/Middlewares/Contexts/DiscordServerContext.cs

```diff
@@ -8,13 +9,15 @@ public class DiscordServerContext : IDiscordContext
         public string Name { get; private set; }
         public UserContext Owner { get; private set; }
         public ChannelContext SystemChannel { get; private set; }
```

Comment:
co to SystemChannel?

Reply:
![image](https://user-images.githubusercontent.com/35143425/74473821-91721f80-4ea4-11ea-85f6-0dd2e02750ba.png)

Reply:
Na nim wita się nowego usera

Reply:
nazwałbym to bardziej jako WelcomeChannel albo LandingChannel